### ISSUE TITLE
Can you share a pretrained model file?

### ISSUE THREAD

**Issue by kafan1986** [TallyH0/RMNet_tensorflow#1]
Can you share the pre-trained model file and what any idea about the accuracy of the model?

**Comment by TallyH0** [comment]
i will update pre-trained model soon.

about accuracy, there are rank1-acc and mAP(mean average preicison).
